src/figures/figureW.jl

```diff
@@ -130,22 +147,34 @@ function plotSynergy(fit::fitResult; L0, f, murine::Bool, c1q = false, neutraliz
     ActI = murine ? murineActI : humanActI
 
     nPoints = 100
-    IgGC = zeros(Float64, size(Kav, 1), nPoints)
-
     M = zeros(size(Kav)[1], size(Kav)[1])
 
     for i = 1:size(Kav)[1]
         for j = 1:(i - 1)
             IgGC = zeros(Float64, size(Kav, 1), nPoints)
+            IgGC[j, :] = range(eps(), eps(); length = nPoints)
```

Comment:
This is just filling with a constant value, right? You can do that with `IgGC[j, :] .= eps()`